test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlAttributeTest.cs

```diff
@@ -29,6 +29,39 @@ public void SimpleLiteralAttribute()
         }
 
         [Fact]
+        public void SimpleLiteralAttributeWithWhitespaceSurroundingEquals()
+        {
+            ParseBlockTest($"<a href \f{Environment.NewLine}= \t{Environment.NewLine}'Foo' />",
```

Comment:
Use of `Environment.NewLine` instead of `\r\n` means you're not testing `'\r'` on Linux. I.e. I suggest being explicit about the characters and dropping `Environment.NewLine`. That should also fix the problems testing on Linux because the source locations will be identical.